src/main/java/de/tum/in/www1/artemis/service/connectors/jenkins/JenkinsService.java

```diff
@@ -250,6 +251,13 @@ public void deleteProject(String projectKey) {
         try {
             jenkinsServer.deleteJob(projectKey, useCrumb);
         }
+        catch (HttpResponseException e) {
+            // We don't throw an exception if the project doesn't exist in Jenkins (404 status)
+            if (e.getStatusCode() != org.apache.http.HttpStatus.SC_NOT_FOUND) {
```

Comment:
you could optimise the import here ;)

Reply:
Sadly it won't work here because the class uses another `HttpStatus` from the Spring package